examples/bri-3/src/bri/identity/bpiSubjects/models/bpiSubject.ts

```diff
@@ -1,24 +1,59 @@
-import { BpiSubjectType } from "./bpiSubjectType.enum";
+import { BpiSubjectType } from './bpiSubjectType.enum';
 
 export class BpiSubject {
-    // Fields of the domain model are always changed through it's methods
-    private id: string; // TODO: Add uuid after #491
-    private name: string;
-    private description: string;
-    private type: BpiSubjectType;
-    private publicKey: string;
+  // Fields of the domain model are always changed through it's methods
+  private _id: string; // TODO: Add uuid after #491
+  private _name: string;
+  private _description: string;
+  private _type: BpiSubjectType;
+  private _publicKey: string;
 
   constructor(
-    id: string, 
-    name: string, 
-    description: string, 
-    type: BpiSubjectType, 
-    publicKey: string
-) {
-    this.id = id
-    this.name = name
-    this.description = description
-    this.type = type
-    this.publicKey = publicKey
-  } 
+    id: string,
+    name: string,
+    description: string,
+    type: BpiSubjectType,
+    publicKey: string,
+  ) {
+    this.id = id;
+    this.name = name;
+    this.description = description;
+    this.type = type;
+    this.publicKey = publicKey;
+  }
+
+  public get id(): string {
+    return this._id;
+  }
+  public set id(value: string) {
```

Comment:
why public setters?

Reply:
good point, added them on autopilot (not github autopilot :)). Removing..